EasyReflectometryApp/Gui/Components/BasePlot.qml

```diff
@@ -54,10 +60,14 @@ Rectangle {
     property color measuredAreaColor: measuredLineColor
     property color calculatedLineColor: EaStyle.Colors.chartForegrounds[1]
     property color sldLineColor: EaStyle.Colors.chartForegrounds[2]
+    property color backgroundLineColor: EaStyle.Colors.chartForegrounds[3]
+    property color scaleLineColor: EaStyle.Colors.chartForegrounds[4]
```

Comment:
Is there item 4 in the `EaStyle.Colors.chartForegrounds` list?

Reply:
The lines look to be the same colour (but I can't tell the difference). It doesn't matter though cause they are visually distinct due to different dashing.